app/src/main/res/values-de/strings.xml

```diff
@@ -77,8 +77,8 @@
     <string name="pref_auto_install_summary_off">Apps werden installiert wenn die Abschluss-Benachrichtigung angetippt wird</string>
     <string name="pref_auto_install_summary_on">Apps werden installiert sobald der Download abgeschlossen ist</string>
     <string name="pref_auto_install">Apps sofort installieren.</string>
-    <string name="pref_update_list_white_or_black_black">Nur für Ausgewählte Apps werden keine Aktualisierungen gesucht</string>
-    <string name="pref_update_list_white_or_black_white">Nur für Ausgewählte Apps werden Aktualisierungen gesucht</string>
+    <string name="pref_update_list_white_or_black_black">Nur für ausgewählte Apps werden keine Aktualisierungen gesucht</string>
+    <string name="pref_update_list_white_or_black_white">Nur für ausgewählte Apps werden Aktualisierungen gesucht</string>
```

Comment:
If we can agree on "Updates" I'll change it. Otherwise I'll change all the "Updates" to "Aktualisierungen" :wink: